x/bep3/legacy/v0_10/types.go

```diff
@@ -0,0 +1,81 @@
+package v0_10
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+)
+
+// GenesisState - all bep3 state that must be provided at genesis
+type GenesisState struct {
+	Params      Params      `json:"params" yaml:"params"`
+	AtomicSwaps AtomicSwaps `json:"atomic_swaps" yaml:"atomic_swaps"`
+}
+
+// Params governance parameters for the bep3 module
+type Params struct {
+	AssetParams AssetParams `json:"asset_params" yaml:"asset_params"`
+}
+
+// AssetParam parameters that must be specified for each bep3 asset
+type AssetParam struct {
+	Denom         string         `json:"denom" yaml:"denom"`                                     // name of the asset
+	CoinID        int            `json:"coin_id" yaml:"coin_id"`                                 // SLIP-0044 registered coin type - see https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+	SupplyLimit   AssetSupply    `json:"supply_limit" yaml:"supply_limit"`                       // asset supply limit
```

Comment:
Ah, the name is outdated now that we released v10. It should be v11 now. It will be used when we write the 'migrate' command to go from kava-3 to kava-4.